hibernate-core/src/main/java/org/hibernate/query/sqm/internal/SqmCriteriaNodeBuilder.java

```diff
@@ -923,12 +937,123 @@ public <N extends Number> SqmExpression<N> abs(Expression<N> x) {
 		);
 	}
 
+	@Override
+	public JpaExpression<Duration> duration(long magnitude, TemporalUnit unit) {
```

Comment:
Or alternatively name this `interval` if we go with `addInterval`/`subtractInterval`

Reply:
It's not an interval man. An interval has fixed endpoints. The people who wrote the ANSI SQL spec just don't know what words mean.